src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -175,7 +175,7 @@
     </PropertyGroup>
 
     <!-- verify the test archives were created -->
-    <Error Condition="'@(ForUpload->Count())' == '0'" Text="Didn't find any test archives in supplied folders." />
+    <Warning Condition="'@(ForUpload->Count())' == '0'" Text="Didn't find any test archives in supplied folders!!! Review logs to see where test archive(s) were placed and fix." />
```

Comment:
I guess this will work for now, but ultimately what we want is to have the folder where we drop the zip files to be static, so that the archives that got created are always uploaded.